ginga/misc/plugins/MultiDim.py

```diff
@@ -316,38 +318,8 @@ def close(self):
         self.fv.stop_local_plugin(self.chname, str(self))
         return True
 
-    def redo(self, channel, image):
-
-        fitsimage = channel.fitsimage
-
-        if fitsimage != self.fitsimage:
-            # Focus is not our channel-->not an event for us
-            return False
-
-        if image is None:
-            return False
-
-        auto_start = self.settings.get('auto_start_naxis', False)
-
-        if not hasattr(image, 'naxispath'):
-            return False
-        # Start ourselves if file is multidimensional
-        if len(image.naxispath) <= 0:
-            return False
-
-        # check preference for auto_start_naxis
-        if not auto_start:
-            return False
-
-        # Start ourselves if GUI is not up yet
-        if not self.gui_up:
-            self.fv.start_local_plugin(self.chname, str(self), None)
-
-        return True
-
-
```

Comment:
@pllim, why remove `redo()`?  It means that `MultiDim` does not reinitialize properly if a new image is set in the channel.  It will display incorrect information, and it will contain the previous image's open `fits_f`.